webapp/utils/utils.jsx

```diff
@@ -69,7 +69,9 @@ export function isMobileApp() {
     const isApi = userAgent.indexOf('Mattermost') !== -1;
     const isShim = userAgent.indexOf('iPhone') !== -1 && userAgent.indexOf('Safari') === -1 && userAgent.indexOf('Chrome') === -1;
 
-    return isApi || isShim;
+    const isDesktopApp = userAgent.indexOf('Electron') !== -1;
```

Comment:
It uses "Electron/version Mattermost/verson" so I think that's fine. I can check out the mobile app's useragent to see if I can redo this function so that it makes more sense.